python/semantic_kernel/connectors/memory/mongodb_atlas/mongodb_atlas_memory_store.py

```diff
@@ -44,6 +46,9 @@ def __init__(
         self._mongo_client = motor_asyncio.AsyncIOMotorClient(
             connection_string or mongodb_atlas_settings_from_dot_env(),
             read_preference=read_preference,
+            driver=DriverInfo(
+                "Microsoft Semantic Kernel", metadata.version("semantic-kernel")
```

Comment:
I need an example of a full driver_name string that results from this statement in order to be able to establish usage analytics when it goes live. Could anyone provide it?

Reply:
If I understand your request correctly: 
In this change, the driver name should come out something akin to: 
```
"driver_name": "PyMongo | Motor | Microsoft Semantic Kernel",
"version": "4.6 | 3.3.x | ...",
```

Reply:
ok this is sufficient, thank you